src/Discord.Net.Core/Entities/Messages/IUserMessage.cs

```diff
@@ -23,7 +23,7 @@ public interface IUserMessage : IMessage
         /// <summary> Removes all reactions from this message. </summary>
         Task RemoveAllReactionsAsync(RequestOptions options = null);
         /// <summary>  Gets all users that reacted to a message with a given emote </summary>
-        Task<IReadOnlyCollection<IUser>> GetReactionUsersAsync(IEmote emoji, int limit = 100, ulong? afterUserId = null, RequestOptions options = null);
+        IAsyncEnumerable<IReadOnlyCollection<IUser>> GetReactionUsersAsync(IEmote emoji, int limit = 100, ulong? afterUserId = null, RequestOptions options = null);
```

Comment:
Make limit required and remove afterUserId

Reply:
It looks like GetMessagesAsync is the only one where we have a default: everywhere else limit is required (and even in IMessageChannel, limit is required!) or is not present at all.